acceptance_tests/test_delete_account.py

```diff
@@ -13,5 +13,6 @@ def test_account_Delete(self):
         response = self.client.post('/', {'email': self.user.email, 'password': self.user.password})
         self.assertEqual(response.url, '/home/')
 
+        resp= self.client.post(f'/deleteAccount/{self.user.user_id}')
+        self.assertEqual(resp.url,'/deleteAccount/1/')
         MyUserModel.objects.get(user_id=1).delete()
```

Comment:
don't delete with the model, it should be getting deleted after the post and you should fail when trying to get that user